### ISSUE TITLE
DownsamplerBlock

### ISSUE THREAD

**Issue by An-Pan** [Eromera/erfnet_pytorch#38]
why there is " self.conv = nn.Conv2d(ninput, noutput-ninput, (3, 3), stride=2, padding=1, bias=True)"
(noutput-ninput, instead of noutput)
And as a result , the network will be  (conv): Conv2d(3, 13, kernel_size=(3, 3), stride=(2, 2), padding=(1, 1)). It's different the paper.

**Comment by Eromera** [comment]
Hi @An-Pan, its not different to the paper, the downsampler block has a 3x3 convolution and a 2x2 max pooling.

**Comment by An-Pan** [comment]
> Hi @An-Pan, its not different to the paper, the downsampler block has a 3x3 convolution and a 2x2 max pooling.

Thanks! I did't notice the torch.cat() function.
